easybuild/easyconfigs/l/Lmod/Lmod-6.3.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = "Lmod"
+version = "6.3"
```

Comment:
recommendation for 6.3.7, several bugfixes landed there!

thanks for the contrib! together with a trick like this, it brings down eb's dependencies from 3 to 2: ;-) 
https://github.com/fgeorgatos/easybuild.experimental/tree/master/users/fgeorgatos/eb_boot